testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -79,6 +79,20 @@
   end
 end
 
+When(/^I wait at most "([^"]*)" seconds until I do not see "([^"]*)" text$/) do |seconds, text|
+  next if has_no_text?(text, wait: 3)
+  repeat_until_timeout(message: "I still see text '#{text}'", timeout: seconds.to_i) do
+    break if has_no_text?(text, wait: 3)
+    begin
```

Comment:
I think this whole `begin`...`end`block is unneeded:
* `accept_prompt` will never be executed since its `do` block is empty
   (if I understand correctly https://www.rubydoc.info/github/jnicklas/capybara/Capybara%2FSession:accept_prompt)
* for the same reason, the `ModalNotFound` exception will never be triggered

I suggest you simply remove the whole block.

Reply:
ok I will try it

Reply:
done , works